object/object.go

```diff
@@ -490,6 +490,29 @@ func (o *Object) SetChildren(v ...oid.ID) {
 	})
 }
 
+// todo
+func (o *Object) SetInitID(id oid.ID) {
+	var v2 refs.ObjectID
+	id.WriteToV2(&v2)
+
+	o.setSplitFields(func(split *object.SplitHeader) {
+		split.SetInit(&v2)
+	})
+}
+
+// todo
+func (o *Object) InitID() (v oid.ID, isSet bool) {
```

Comment:
I suggest replacing `isSet` with `error`. Anyway, we return the second variable. The error is more informative and sometimes required to understand what it is

Reply:
agree it can be improved somehow but suggest doing it all together (these flags are [common](https://github.com/nspcc-dev/neofs-sdk-go/blob/1db2fbf3c1ad047d1ab3dd74d7c3a930856cf0f0/object/object.go#L235) to the `Object`)